src/helpers/validation.js

```diff
@@ -89,9 +89,6 @@ export const validateProjectType = (projectType) => {
     return "Project Type should start with a letter";
   }
 
-  if (validateName(projectType) === "false_convention") {
```

Comment:
@khalifan-kfan did you remove this function because we actually do use it.

Reply:
Alright sir

Reply:
This condition is checked twice in the function @neelxie check line 84